app/models/UserAnswers.scala

```diff
@@ -32,9 +33,9 @@ import queries.Modifiable
 final case class UserAnswers(
   id: String,
   applicationNumber: String,
+  affinityGroup: AffinityGroup,
```

Comment:
We do have affinityGroup living on the request so far, should we move to only use it on the UserAnswers or just keep both? 🤔

Reply:
Good suggestion. I think having it in the UserAnswers may not be required. I'll fix now.